compiler/src/main/scala/dx/translator/cwl/ProcessTranslator.scala

```diff
@@ -575,24 +575,14 @@ case class ProcessTranslator(cwlBundle: CwlBundle,
         param.name -> param.dxType
       }.toMap
 
-      val (instanceType, container, attributes, requirements) = availableDependencies
-        .collectFirst({
-          case (iname, i: Application) if iname == calleeName.getOrElse("") => {
-            (i.instanceType, i.container, i.attributes, i.requirements)
-          }
-        })
-        .getOrElse((DefaultInstanceType, NoImage, Vector.empty, Vector.empty))
```

Comment:
PR#326 only covers the case in which the `callee` in cwl or the `innerCall` in wdl is an application. If the inner callable of a frag applet is a subworkflow (which does not have `container` attr), the frag applet would not be given network access during compilation. 

So here I would just revert this change, and in `ApplicationCompiler` the frag applet will still be given full access.